### ISSUE TITLE
fairseq is missing in v.1.5.1.8.

### ISSUE THREAD

**Issue by hto0204** [w-okada/voice-changer#142]
v.1.5.1.8(ONNX(cpu,DirectML), PyTorch(cpu,cuda)) is missing fairseq so GUI is not activated in this version.
In ONNX(cpu,cuda), PyTorch(cpu,cuda), I have confirmed that GUI is running and fairseq is included.
Here is the error code.

```
Traceback (most recent call last):
  File "multiprocessing\process.py", line 314, in _bootstrap
  File "multiprocessing\process.py", line 108, in run
  File "MMVCServerSIO.py", line 112, in localServer
  File "uvicorn\main.py", line 569, in run
  File "uvicorn\server.py", line 60, in run
  File "asyncio\runners.py", line 44, in run
  File "asyncio\base_events.py", line 649, in run_until_complete
  File "uvicorn\server.py", line 67, in serve
  File "uvicorn\config.py", line 477, in load
  File "uvicorn\importer.py", line 24, in import_from_string
  File "uvicorn\importer.py", line 21, in import_from_string
  File "importlib\__init__.py", line 126, in import_module
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "PyInstaller\loader\pyimod02_importers.py", line 352, in exec_module
  File "MMVCServerSIO.py", line 125, in <module>
  File "voice_changer\VoiceChangerManager.py", line 10, in get_instance
  File "voice_changer\VoiceChanger.py", line 59, in __init__
  File "PyInstaller\loader\pyimod02_importers.py", line 352, in exec_module
  File "voice_changer\SoVitsSvc40v2\SoVitsSvc40v2.py", line 24, in <module>
ModuleNotFoundError: No module named 'fairseq'
```

**Comment by w-okada** [comment]
Thanks,　I'll check it.

**Comment by w-okada** [comment]
I fixed it with version 1.5.1.9. If you have a chance, please check it out.

**Comment by w-okada** [comment]
no response. close
